### ISSUE TITLE
Parsing of `def` with flags fails

### ISSUE THREAD

**Issue by kit494way** [nushell/tree-sitter-nu#49]
![スクリーンショット 2023-12-27 221408](https://github.com/nushell/tree-sitter-nu/assets/17674842/ff8d632c-9453-432f-bc21-f5ea3c57a1bd)

**Comment by fdncred** [comment]
right, this is relatively new. we have `def --env` and `def --wrapped` right now and we could have `def --env --wrapped` or `def --wrapped --env`.
